eahub/config/settings.py

```diff
@@ -193,11 +193,16 @@ class DjangoEnv(Enum):
 ACCOUNT_USERNAME_REQUIRED = False
 
 # search
+IS_ENABLE_ALGOLIA = True
```

Comment:
I would recommend `env.str("IS_ENABLE_ALGOLIA", default=True)` and then an addition to `.env` as `IS_ENABLE_ALGOLIA=false`, since sounds as we can benefit from running the tests with agolia through github actions? and I personally would run them locally as well

Reply:
Implemented.